components/api_version/src/api_v2.rs

```diff
@@ -235,6 +235,16 @@ impl ApiV2 {
         Ok(Key::split_on_ts_for(key)?)
     }
 
+    pub fn add_prefix(key: &[u8]) -> Vec<u8> {
+        let mut apiv2_key = Vec::with_capacity(ApiV2::get_encode_len(key.len() + 2));
```

Comment:
Reserve capacity after encoded keyspace id is got and avoid hard coding the length.

Reply:
done